txp/Pos/Txp/Toil/Utxo/Util.hs

```diff
@@ -33,3 +35,11 @@ utxoToStakes = foldl' putDistr mempty . M.toList
   where
     plusAt hm (key, val) = HM.insertWith unsafeAddCoin key val hm
     putDistr hm (_, toaux) = foldl' plusAt hm (txOutStake toaux)
+
+utxoToAddressCoinPairs :: Utxo -> [(Address, Coin)]
```

Comment:
This reasoning is quite good. To inform the person looking at this code that the keys are unique is a good idea.